chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/RoomEventParserSpec.kt

```diff
@@ -0,0 +1,79 @@
+package com.pusher.chatkit.rooms
+
+import com.google.common.truth.Truth.assertThat
+import com.pusher.chatkit.TestFileReader
+import com.pusher.chatkit.rooms.api.CreateRoomResponse
+import com.pusher.chatkit.rooms.api.JoinRoomResponse
+import com.pusher.chatkit.rooms.api.JoinedRoomApiType
+import com.pusher.chatkit.util.parseAs
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+object RoomEventParserSpec : Spek({
+    val testFileReader = TestFileReader("/json/rooms")
+
+    describe("when create room response from the docs is parsed") {
+
+        val result = testFileReader.readTestFile("room_created_response-docs.json")
```

Comment:
How about this [bit of my previous feedback](https://github.com/pusher/chatkit-android/pull/150#discussion_r370249398)? It would be good to use consistent naming with how we call the endpoint in the code and how it's called in the backend docs.

Reply:
I used the GitHub page to commit that change as you requested which made `create_room-docs.json` to `create_room_response-docs.json` - I had to update both the file names. Is that not what you had in mind?

Reply:
It hasn't been committed. I further [suggested](https://github.com/pusher/chatkit-android/pull/150#discussion_r371794011) merging the jsons as they are the same (indeed also semantically creating a room means joining it).